apps/home/app/_shared/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,49 @@
+import { Email, People } from '@hamsurang/icon'
+import { Avatar, AvatarFallback, AvatarImage } from '@hamsurang/ui'
+
+type ProfileProps = {
+  name: string
+  username: string
+  email: string
+  description?: string
+  followers?: number
+}
+
+export const Profile = ({ name, username, description, email, followers }: ProfileProps) => {
+  return (
+    <div>
+      <div className="flex flex-col mobile:flex-row mobile:items-center mobile:gap-4 mobile:pb-4">
+        <Avatar className="w-[296px] h-[296px] mobile:size-[16%] mobile:aspect-square">
+          <AvatarImage
+            src={`https://github.com/${username}.png?size=296`}
+            alt={`${username}의 프로필`}
+          />
+          <AvatarFallback />
+        </Avatar>
+
+        <div className="flex flex-col py-4 mobile:flex-grow mobile:py-0">
+          <span className="text-3xl font-semibold mobile:text-lg">{name}</span>
+          <span className="text-gray-500">{username}</span>
+          {description && <p>{description}</p>}
+        </div>
+      </div>
+
+      <div className="flex gap-4 text-gray-500 mobile:flex-col mobile:gap-2 mobile:text-sm">
+        {followers && (
```

Comment:
```suggestion
        {!!followers && (
```
number 타입이어서 boolean으로 형변환해야 의도대로 동작할 것 같습니다!

Reply:
undefine 이 가능한 요소라 형변환이 아니라 명시되면 표기하려고 한건데
형변환이 필요할까요? 멤버 페이지에서는 follower 표기가 아예없어서요
<img width="495" alt="image" src="https://github.com/user-attachments/assets/dfdaf5bd-64d0-4735-a778-bdbc695b1302">